moped-editor/src/views/projects/projectView/ProjectNotes/NoteTypeRadioButtons.js

```diff
@@ -0,0 +1,30 @@
+import { Radio, RadioGroup, FormControlLabel } from "@mui/material";
+
+/**
+ * Defines the NoteTypeRadioButtons that allow a user to select the note type of a
+ * note they are creating or editing.
+ * @param {integer} defaultValue - The note type id that determines which button option
+ * should be selected/highlighted by default
+ * @param {function} onChange - Callback function that runs when the note type is changed via button click
```

Comment:
could you add the `noteTypes` param to the jsdoc? cool to see these radios rendering from DB data now!